viite-backend/viite-main/src/test/scala/fi/liikennevirasto/viite/ProjectServiceSpec.scala

```diff
@@ -4796,8 +4796,14 @@ class ProjectServiceSpec extends AnyFunSuite with Matchers with BeforeAndAfter {
       val (terminated, others) = recalculated.partition(_.status == RoadAddressChangeType.Termination)
       val leftTerminated = terminated.filter(_.track == Track.LeftSide)
       val rightTerminated = terminated.filter(_.track == Track.RightSide)
-      leftTerminated.head.addrMRange.end should be (21)
-      rightTerminated.head.addrMRange.end should be (21)
+      val newLeftStartingLink = others.filter(pl => pl.addrMRange.isRoadPartStart && pl.track == Track.LeftSide)
+      val newRightStartingLink = others.filter(pl => pl.addrMRange.isRoadPartStart && pl.track == Track.RightSide)
+      newLeftStartingLink.size should be (1)
```

Comment:
Vaihtaisin terminoitujen testit ennen jäljelle jääneiden testejä - tässä järjestyksessä niiden prosessointi laskennassakin menee?
Ja vielä kommentit. Terminoiduille:
// After termination, the terminated parts' addresses should be averaged
Ja jäljelle jääneille:
// And the remaining links, continuing after termination, should have the same averaged address as the terminated got at end

tai vastaavaa